R-package/R/lightgbm.R

```diff
@@ -92,6 +92,13 @@ NULL
 #' @inheritParams lgb_shared_params
 #' @param label Vector of labels, used if \code{data} is not an \code{\link{lgb.Dataset}}
 #' @param weight vector of response values. If not NULL, will set to dataset
+#' @param init_score Initial values for each observation from which the boosting process will
```

Comment:
Please add a new test that passing `init_score` to `lightgbm::lightgbm()` works as expected. Any user-facing changes to behavior should be accompanied by unit tests to ensure that the functionality is working as expected and to reduce the risk that future changes accidentally break that functionality.